platform/ui/src/components/StudyBrowser/StudyBrowser.tsx

```diff
@@ -140,7 +140,7 @@ StudyBrowser.propTypes = {
               imageSrc: PropTypes.string,
               imageAltText: PropTypes.string,
               seriesDate: PropTypes.string,
-              seriesNumber: PropTypes.string,
+              seriesNumber: PropTypes.any,
```

Comment:
How can `seriesNumber` be any? isn't it `number` or `string`? `any` means it can be an object or a function too which is wrong